warning in cm/uct_p2p_am_test.am_bcopy - got active message id 11, but no handler installed

http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7746/label=r-vmb-ppc-jenkins,worker=1/consoleFull
```
20:56:00 [ RUN      ] cm/uct_p2p_am_test.am_bcopy/0
20:56:00 [     INFO ] memory_type:host 0 214 1000x{0..214} [1537811762.740187] [r-vmb-ppc-jenkins:15036:0]      uct_iface.c:54   UCX  WARN  got active message id 11, but no handler installed
20:56:02 nocomp 14 
20:56:02 /scrap/jenkins/workspace/hpc-ucx-pr/label/r-vmb-ppc-jenkins/worker/1/contrib/../test/gtest/common/test.cc:264: Failure
20:56:02 Failed
20:56:02 Got 3 warnings during the test
20:56:02 
[  FAILED  ] cm/uct_p2p_am_test.am_bcopy/0, where GetParam() = cm/mlx5_0:1 (2109 ms)
...
21:11:34 [  PASSED  ] 2734 tests.
21:11:34 [  FAILED  ] 1 test, listed below:
21:11:34 [  FAILED  ] cm/uct_p2p_am_test.am_bcopy/0, where GetParam() = cm/mlx5_0:1
21:11:34 
21:11:34  1 FAILED TEST
21:11:34 
make: *** [test] Error 1
21:11:34 make: Leaving directory `/scrap/jenkins/workspace/hpc-ucx-pr/label/r-vmb-ppc-jenkins/worker/1/build-test/test/gtest'
21:11:34 
```

--- a/test/gtest/uct/test_p2p_am.cc
+++ b/test/gtest/uct/test_p2p_am.cc
@@ -407,7 +407,7 @@ public:
 
 
 UCS_TEST_P(uct_p2p_am_test, am_short) {
-    check_caps(UCT_IFACE_FLAG_AM_SHORT);
+    check_caps(UCT_IFACE_FLAG_AM_SHORT, UCT_IFACE_FLAG_AM_DUP);
     test_xfer_multi(static_cast<send_func_t>(&uct_p2p_am_test::am_short),
                     sizeof(uint64_t),
                     sender().iface_attr().cap.am.max_short,
@@ -415,7 +415,7 @@ UCS_TEST_P(uct_p2p_am_test, am_short) {
 }
 
 UCS_TEST_P(uct_p2p_am_test, am_bcopy) {
-    check_caps(UCT_IFACE_FLAG_AM_BCOPY);
+    check_caps(UCT_IFACE_FLAG_AM_BCOPY, UCT_IFACE_FLAG_AM_DUP);
     test_xfer_multi(static_cast<send_func_t>(&uct_p2p_am_test::am_bcopy),
                     0ul,
                     sender().iface_attr().cap.am.max_bcopy,
@@ -432,7 +432,7 @@ UCS_TEST_P(uct_p2p_am_test, am_short_keep_data) {
 }
 
 UCS_TEST_P(uct_p2p_am_test, am_bcopy_keep_data) {
-    check_caps(UCT_IFACE_FLAG_AM_BCOPY);
+    check_caps(UCT_IFACE_FLAG_AM_BCOPY, UCT_IFACE_FLAG_AM_DUP);
     set_keep_data(true);
     test_xfer_multi(static_cast<send_func_t>(&uct_p2p_am_test::am_bcopy),
                     sizeof(uint64_t),
@@ -441,7 +441,7 @@ UCS_TEST_P(uct_p2p_am_test, am_bcopy_keep_data) {
 }
 
 UCS_TEST_P(uct_p2p_am_test, am_zcopy) {
-    check_caps(UCT_IFACE_FLAG_AM_ZCOPY);
+    check_caps(UCT_IFACE_FLAG_AM_ZCOPY, UCT_IFACE_FLAG_AM_DUP);
     test_xfer_multi(static_cast<send_func_t>(&uct_p2p_am_test::am_zcopy),
                     0ul,
                     sender().iface_attr().cap.am.max_zcopy,

